core/domain/suggestion_registry_test.py

```diff
@@ -1892,10 +1912,11 @@ def test_validate_change_question_dict(self) -> None:
 
         suggestion.validate()
 
-        # Here, change is of type QuestionSuggestionChange and all attributes
-        # on QuestionSuggestionChange are created dynamically except cmd, so due
-        # this MyPy is unable to recognize `question_dict` as an attribute of
-        # change and throwing `"ExplorationChange" has no attribute "skill_id"`
+        # Here we use MyPy ignore because change is of type
```

Comment:
Please double-check this and the other comments below. I'm not sure the rationale is correct here, per earlier comments.

Reply:
This issue is already fixed in https://github.com/oppia/oppia/pull/15958